primefaces/src/main/resources/META-INF/resources/primefaces/avatar/avatar.css

```diff
@@ -24,6 +24,10 @@
     height: 100%;
 }
 
-.ui-avatar .ui-avatar-dynamic {
-    mix-blend-mode: difference;
-}
\ No newline at end of file
+.ui-avatar.ui-avatar-dark {
```

Comment:
@melloware @mertsincan @sqores maybe this styling of dynamic background text colors should go to the themes. WDYT?

Reply:
I think for now defaults are OK here and they can be overridden in themes if necessary.